applications/RomApplication/custom_strategies/rom_builder_and_solver.h

```diff
@@ -371,150 +356,125 @@ class ROMBuilderAndSolver : public BuilderAndSolver<TSparseSpace, TDenseSpace, T
 	// }
 
     void ProjectToFineBasis(
-        const TSystemVectorType &rRomUnkowns,
-        ModelPart &rModelPart,
-        TSystemVectorType &Dx)
+        const TSystemVectorType& rRomUnkowns,
+        ModelPart& rModelPart,
+        TSystemVectorType& rDx)
     {
-        const auto dofs_begin = BaseType::mDofSet.begin();
-        const auto dofs_number = BaseType::mDofSet.size();
+        auto& r_dof_set = BaseType::GetDofSet();
+        const int dofs_number = r_dof_set.size();
+        const auto dofs_begin = r_dof_set.begin();
 
         #pragma omp parallel firstprivate(dofs_begin, dofs_number)
         {
             const Matrix *pcurrent_rom_nodal_basis = nullptr;
             unsigned int old_dof_id;
             #pragma omp for nowait
-            for (int k = 0; k < static_cast<int>(dofs_number); k++){

```

Comment:
I remember having done this to avoid errors when compiling in Windows. But I assumed it is compatible now, as it is passing the CI...

Reply:
As far as I know the issue with Windows parallelism is still there (`unsigned int` parallel for is not supported). This is why we used to cast to `int`. Note that in this case I'm implicitly casting to `int` when declaring the variable as `const int dofs_number = r_dof_set.size();`.